cartoframes/viz/widget_list.py

```diff
@@ -46,10 +46,16 @@ def _init_widgets(self, widgets):
     def get_variables(self):
         variables = {}
         for widget in self.widgets:
-            if widget and widget.has_variable():
+            if widget and widget._value:
                 variables[widget._name] = widget._value
```

Comment:
Not all widgets require this. Category widget and histogram don't really need to define a variable.

I would make it so has_variable returns true for the animation widget if that's necessary.

Reply:
Also, as it is right now, the category widget and histogram widgets don't accept `value: $propName`, but `value: propName`, so that would end up being invalid CARTO VL syntax.

Maybe we should fix that in the bridge, if whatever comes starts with a $, it's a column name, so we can drop it.

Reply:
Fixed 👌